src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/XmlSerializableServices.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Runtime.Serialization
+{
+    using System.Collections.Generic;
+    using System.Xml;
+    using System.Xml.Schema;
+
+    public static class XmlSerializableServices
+    {
+        internal static readonly string ReadNodesMethodName = "ReadNodes";
```

Comment:
Afaik, ```static readonly``` is more future proof than ```const``` in case we ever want to change it. Const string would be embeded to the assembly and I think const is more suitable for things that will always stay the same such as physics constant. I'll keep this as it is since this is a porting change.